flink/v1.13/flink/src/main/java/org/apache/iceberg/flink/FlinkCatalogFactory.java

```diff
@@ -153,6 +156,17 @@ private static Configuration mergeHiveConf(Configuration hadoopConf, String hive
         newConf.addResource(configFile);
       }
     }
+
+    if (!Strings.isNullOrEmpty(hadoopConfDir)) {
+      if (new File(hadoopConfDir).exists()) {
+        Preconditions.checkState(Files.exists(Paths.get(hadoopConfDir, "hdfs-site.xml")),
+                "There should be a hdfs-site.xml file under the directory %s", hadoopConfDir);
```

Comment:
The indentation is off; please fix this to match the rest of the file.

This error message (and the existing one) aren't very good. In Iceberg, we are clear about what was attempted and what happened, then clarify why. Here, the error message should be `"Failed to load Hadoop configuration: missing %s", hiveSitePath`

Reply:
Such error messages are indeed clearer. should I modify the existing error message in this PR or in another PR.